src/sonic-config-engine/minigraph.py

```diff
@@ -836,7 +837,10 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None):
         if child.tag == str(docker_routing_config_mode_qn):
             docker_routing_config_mode = child.text
 
-    (ports, alias_map, alias_asic_map) = get_port_config(hwsku=hwsku, platform=platform, port_config_file=port_config_file, asic=asic_id)
+    if hwsku_config_file:
+        (ports, alias_map, alias_asic_map) = get_port_config(hwsku=hwsku, platform=platform, port_config_file=port_config_file, hwsku_config_file=hwsku_config_file, asic=asic_id)
```

Comment:
We don't really need an if/else. hwsku will be passes as None, if not there.

Reply:
Yes, correct! 👍  Modified as requested!